### ISSUE TITLE
Can we avoid generating empty JavaScript files?

### ISSUE THREAD

**Issue by MangelMaxime** [fable-compiler/Fable#2868]
### Description

When an F# file is a pure binding Fable generates an empty file.

**CssModules.fs**

```fs
module CssModules

// IMPORTANT: Do not edit this file
// It is generated each time a CSS module is updated.


/// Definition of the Components/Connecting.module.scss CSS module.
type Components_Connecting =

    [<Emit("$0[\"gradient-backdrop-2main\"]")>]
    abstract gradientBackdropMain : string
```

generates

**CssModules.fs.js**

```js

//# sourceMappingURL=CssModules.fs.js.map

```

The problem is that even if the file is empty when it is re-generated it messing up with react refresh at least from vite and it triggers a full reload of the page.

I see two potential fix for this situation:

1. Do not generate the file if it will be empty.

Not generating the file in the first case is probably the easiest part of this request. The hard part is, if the file was generated with actual code but now become a pure binding can we delete the file?

2. Don't update the file if nothing changed. Because, I suppose that if Vite is triggering a full reload this is because Fable updated the file.

### Workaround

Adding some stable code to the file seems to fix the problem.

**CssModules.fs**

```fs
module CssModules

// IMPORTANT: Do not edit this file
// It is generated each time a CSS module is updated.

// If the file is empty, Vite trigger a full refresh of the page.
// Adding a stable constant to the file will prevent this.
let private stabilizeReactRefresh = "";

/// Definition of the Components/Connecting.module.scss CSS module.
type Components_Connecting =

    [<Emit("$0[\"gradient-backdrop-2main\"]")>]
    abstract gradientBackdropMain : string
```

generates

**CssModules.fs.js**

```js

const stabilizeReactRefresh = "";

//# sourceMappingURL=CssModules.fs.js.map
```

### Related information

* Fable version: `3.7.5`
* Operating system: Windows

**Comment by alfonsogarciacaro** [comment]
This shouldn't be difficult. It also bothered me a couple of times but I forgot to fix it in Fable 😅
